src/content/en/2021/webassembly.md

```diff
@@ -1,11 +1,11 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: WebAssembly
-description: TODO
+description: WebAssembly chapter of the 2021 Web Almanac covering Wasm compression, custom sections, most popular instructions, and post-MVP Wasm extensions
```

Comment:
@RReverser noticed we didn't have a Meta Description so added this one. You happy with that?

Reply:
```suggestion
description: WebAssembly chapter of the 2021 Web Almanac covering Wasm compression, section sizes, most popular instructions, and post-MVP proposals
```

maybe a bit better